utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt

```diff
@@ -36,8 +36,8 @@ class FoldableJUnit4ClassRunner : AndroidJUnit4ClassRunner {
         val currentDeviceModel = uiDevice.getDeviceModel()
         val targetDevices: TargetDevices? = getAnnotation(description, TargetDevices::class.java)
         if (targetDevices == null ||
-            targetDevices.devices.any { it == currentDeviceModel } ||
-            !targetDevices.ignoreDevices.any { it == currentDeviceModel }
+            (targetDevices.devices.isNotEmpty() && targetDevices.devices.any { it == currentDeviceModel }) ||
+            (targetDevices.ignoreDevices.isNotEmpty() && !targetDevices.ignoreDevices.any { it == currentDeviceModel })
```

Comment:
These two lines are a bit difficult to read and understand, maybe is a good idea to create their own functions with a name that can be understandable easily and use them?
e.g.
```
if (targetDevices == null ||
   targetDeviceIsCurrentDevice ||
   ignoreDeviceIsNotCurrentDevice) {
   ....
   }
```

Thoughts?

Reply:
Done. Added two functions: currentDeviceIsOnTargetDevices, currentDeviceNotInIgnoredDevices

Reply:
Awesome! thanks Andrei! It looks great now.